### PR TITLE
booking model created and db migrated, foreing keys from user and pet

### PR DESCRIPTION
**Pull request by dacoreggio**
Screenshot 2021-01-26 at 21 11 21
Screenshot 2021-01-26 at 21 10 30

### CODE REVIEW

**Comment by dacoreggio** [comment]
Seems to be ok @fedeval , could you take a look at it?

**Comment by dacoreggio** [comment]
need to add settings to end_date (higher than start_date) and price higher than 0

File: app/models/booking.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Booking < ApplicationRecord
+  belongs_to :user
+  belongs_to :pet
+
+  validates :start_date, presence: true
+  validates :end_date, presence: true
```
Comment (fedeval) [inline app/models/booking.rb:6]: this should be validated to only dates after start_date

File: app/models/booking.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  validates :price, presence: true
```
Comment (fedeval) [inline app/models/booking.rb]: shoud be greater than zero

File: app/models/booking.rb (hunk 1 above)
Comment (dacoreggio) [reply app/models/booking.rb:6]: also add Pandit authorization for bookings

**Pull request merged**